pkg/graveler/graveler.go

```diff
@@ -664,6 +665,8 @@ type RefManager interface {
 	// FillGenerations computes and updates the generation field for all commits in a repository.
 	// It should be used for restoring commits from a commit-dump which was performed before the field was introduced.
 	FillGenerations(ctx context.Context, repositoryID RepositoryID) error
+
+	Store() *kv.StoreMessage
```

Comment:
Why? It's not a feature of graveler.

Reply:
it is needed for initializing the commit iterator on the garbage collection

Reply:
done